### PR TITLE
Provide types for convert-units package

### PR DESCRIPTION
**Pull request by vladkampov**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by vladkampov** [comment]
fyi `convert-units` lib has types but it's seems like it's not maintained by it owner: latest changes with types was in April 2018 and hasn't been released to npm.

File: types/convert-units/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "strictFunctionTypes": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```
Comment (sandersn) [inline types/convert-units/tsconfig.json]: this needs to be false for backward compatibility

File: types/convert-units/convert-units-tests.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import convert from 'convert-units';
```
Comment (sandersn) [inline types/convert-units/convert-units-tests.ts]: Use the commonjs syntax `import convert = require('convert-units')`.

(You can still use `import convert from 'convert-units'` with `"esModuleInterop": true`.)

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/convert-units/tsconfig.json (hunk 1 above)
Comment (vladkampov) [reply types/convert-units/tsconfig.json]: Fixed

File: types/convert-units/convert-units-tests.ts (hunk 2 above)
Comment (vladkampov) [reply types/convert-units/convert-units-tests.ts]: Fixed

**Review by sandersn — APPROVED** [review]

**Pull request merged**